### ISSUE TITLE
missing "logstash/logging/json" require? 

### ISSUE THREAD

**Issue by colinsurprenant** [elastic/logstash#5297]
running bare `bin/rspec logstash-core/spec/logstash/pipeline_spec.rb`  results in a few 

```
LogStash::Pipeline when collecting metrics in the pipeline pipelines populates the pipelines core metrics
     Failure/Error: Thread.new { subject.run }
     NameError:
       uninitialized constant LogStash::Logging
     # ./spec/spec_helper.rb:29:in `(root)'
     # ./vendor/bundle/jruby/1.9/gems/jls-grok-0.11.2/lib/grok-pure.rb:51:in `initialize'
     # ./vendor/bundle/jruby/1.9/gems/logstash-filter-multiline-3.0.1/lib/logstash/filters/multiline.rb:140:in `register'
     # ./logstash-core/lib/logstash/pipeline.rb:216:in `start_workers'
     # ./logstash-core/lib/logstash/pipeline.rb:216:in `start_workers'
     # ./logstash-core/lib/logstash/pipeline.rb:170:in `run'
     # ./logstash-core/spec/logstash/pipeline_spec.rb:576:in `(root)'
```

It seems this is because `logstash/spec/spec_helper.rb` is missing 
```ruby
require "logstash/logging/json"
```

Adding it solves locally.

**Comment by jordansissel** [comment]
oops, yes, `spec/spec_helper.rb` is missing a require.

**Comment by jordansissel** [comment]
Not sure why things were working for me though.. hmm.

**Comment by colinsurprenant** [comment]
I assume it's that when running all specs it works because it gets loaded somewhere else but just running   `pipeline_spec.rb`  in isolation fails.

**Comment by jordansissel** [comment]
Merged fix into 5.0 and master branches. I checked 2.x and it already has this `require` line so we're good. <3
